tools/localization/generateLocales.py

```diff
@@ -138,9 +160,81 @@
+      if "missing_b_tags" in locale_problems:
+        for problem_string, tag_issues in locale_problems["missing_b_tags"].items():
+          if tag_issues > 0:
+            print("ME", problem_string, tag_issues)
+            if problem_string not in string_to_locales:
+              string_to_locales[problem_string] = [locale]
+            else:
+              string_to_locales[problem_string].append(locale)
+      if "missing_span_tags" in locale_problems:
+        for problem_string, tag_issues in locale_problems["missing_span_tags"].items():
+          if tag_issues > 0:
+            if problem_string not in string_to_locales:
+              string_to_locales[problem_string] = [locale]
+            else:
+              string_to_locales[problem_string].append(locale)
+
     console.info(f"Problem strings: {json.dumps(string_to_locales, indent=2)}")
     message += " See above for problem strings and which locales they are in."
 
+  if args.print_problem_formatting_tag_strings:
+    locales_to_strings = {}
+    for locale, locale_problems in problems.items():
+      locale_missing_br_tags = set()
+      locale_missing_b_tags = set()
+      locale_missing_span_tags = set()
+      if "missing_br_tags" in locale_problems:
+        for problem_string, tag_issues in locale_problems["missing_br_tags"].items():
+          if tag_issues > 0:
+            locale_missing_br_tags.add(problem_string)
+      if "missing_b_tags" in locale_problems:
+        for problem_string, tag_issues in locale_problems["missing_b_tags"].items():
+          if tag_issues > 0:
+            locale_missing_b_tags.add(problem_string)
+      if "missing_span_tags" in locale_problems:
+        for problem_string, tag_issues in locale_problems["missing_span_tags"].items():
+          if tag_issues > 0:
+            locale_missing_span_tags.add(problem_string)
+
+      locales_to_strings[locale] = {
+        "br": list(locale_missing_br_tags),
+        "b": list(locale_missing_b_tags),
+        "span": list(locale_missing_span_tags),
+      }
+
+      if locales_to_strings[locale]["br"] == []:
+        del locales_to_strings[locale]["br"]
+      if locales_to_strings[locale]["b"] == []:
+        del locales_to_strings[locale]["b"]
+      if locales_to_strings[locale]["span"] == []:
+        del locales_to_strings[locale]["span"]
+
+    console.info(f"Problem strings: {json.dumps(locales_to_strings, indent=2)}")
+    message += " See above for problem strings and which locales they are in."
+    for locale, locale_strings in locales_to_strings.items():
+      printed_locale = False
+      for tag_type, tag_strings in locale_strings.items():
+        if tag_strings:
+          if locale in ignored_strings_formatting and tag_strings == ignored_strings_formatting[locale]:
+            continue
+          if not printed_locale:
+            print(f"{locale} - [Link Here](https://crowdin.com/editor/session-crossplatform-strings/300/en-{locale})")
```

Comment:
@Aerilym Just a long term thought but might be good to add support for locales on the getsession.org/translate e.g. `https://getsession.org/translate?locale=en`